doc/source/virtual-cluster/cli.rst

```diff
@@ -0,0 +1,401 @@
+    :header-rows: 1
+
+    * - Option
+      - Type
+      - Default
+      - Required
+      - Description
+    * - --address TEXT
+      - str
+      - None
+      - NO
+      - Specifies the Ray cluster address. If not provided, the **RAY_ADDRESS** environment variable is used.
+    * - --id TEXT
+      - str
+      - N/A
+      - YES
+      - Assigns a unique identifier to the Virtual Cluster being created.
+    * - --divisible
+      - bool
+      - False
+      - NO
+      - Determines if the Virtual Cluster is divisible into smaller logical or job clusters.
+    * - --replica-sets TEXT
+      - dict
+      - N/A
+      - YES
+      - JSON-serialized dictionary defining the replica sets for the cluster (e.g., `{"group1":2,"group2":3}`, `group1` and `group2` correspond to the node type names passed via the `RAY_NODE_TYPE_NAME` environment variable).
+
+Usage Examples
+~~~~~~~~~~~~~~
+
+**Example:** Creating a Divisible Virtual Cluster
+
+`ray vcluster create --id myCluster --divisible --replica-sets '{"group1":2,"group2":3}'`
+
+**Output:**
+
+`Virtual cluster 'myCluster' created successfully`
+
+Update a Virtual Cluster
+------------------------
+
+**Description:**
+
+Update an existing Virtual Cluster with specified parameters.
+
+**Command:**
+
+`ray vcluster update [OPTIONS]`
+
+**Options:**
+
+.. list-table::
+    :widths: 20 16 16 16 60
+    :header-rows: 1
+
+    * - Option
+      - Type
+      - Default
+      - Required
+      - Description
+    * - --address TEXT
+      - str
+      - None
+      - NO
+      - Specifies the Ray cluster address. If not provided, the **RAY_ADDRESS** environment variable is used.
+    * - --id TEXT
+      - str
+      - N/A
+      - YES
+      - Assigns a unique identifier to the Virtual Cluster being created.
+    * - --divisible
+      - bool
+      - False
+      - NO
+      - Determines if the Virtual Cluster is divisible into smaller logical or job clusters.
+    * - --replica-sets TEXT
+      - dict
+      - N/A
+      - YES
+      - JSON-serialized dictionary defining the replica sets for the cluster (e.g., `{"group1":2,"group2":3}`, `group1` and `group2` correspond to the node type names passed via the `RAY_NODE_TYPE_NAME` environment variable).
+    * - --revision INTEGER
+      - int
+      - 0
+      - NO
+      - Indicates the revision number for updating the Virtual Cluster.
+
+Usage Examples
+~~~~~~~~~~~~~~
+
+**Example 1:** Updating a Divisible Virtual Cluster
+
+`ray vcluster update --id myCluster --divisible --replica-sets '{"group1":2,"group2":3}'`
+
+**Output:**
+
+`Virtual cluster 'myCluster' updated successfully`
+
+**Example 2:** Handling Updating Failure Due to Incorrect Revision
+
+`ray vcluster update --id myCluster --divisible --replica-sets '{"group1":2,"group2":3}' --revision 2`
+
+**Output:**
+
+`Failed to update virtual cluster 'myCluster': The revision (2) is expired, the latest revision of the virtual cluster myCluster is 1736911613521214948`
```

Comment:
Failed to update virtual cluster 'logical1': The revision (2) is expired, the latest revision of the virtual cluster logical1 is 1736911613521214948